lib/l10n/app_id.arb

```diff
@@ -445,7 +445,7 @@
   "settingsThumbnailShowTagIcon": "Tampilkan ikon label",
   "settingsThumbnailShowLocationIcon": "Tampilkan ikon lokasi",
   "settingsThumbnailShowMotionPhotoIcon": "Tampilkan ikon Foto bergerak",
-  "settingsThumbnailShowRating": "Tampilkan peringkat",
+  "settingsThumbnailShowRating": "Tampilkan ",
```

Comment:
Didn't you miss "nilai"?

Reply:
my text editor decided to remove "subtitle" on every line,filled it again
guessed i missed one